templates/focus-ios.xliff

```diff
@@ -269,13 +396,19 @@
         <source>Settings</source>
         <note>Title for settings screen</note>
       </trans-unit>
+      <trans-unit id="Settings.searchLabel">
+        <source>Search Engine</source>
+        <note>Label for the search engine in the search screen</note>
+      </trans-unit>
       <trans-unit id="Settings.searchSection">
         <source>SEARCH ENGINE</source>
-        <note>Title for the search engine row</note>
```

Comment:
Wondering why we're dropping the localization note in this case? is this expected?